src/constants.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+// App constants
+
+export const REACT_TOTAL_NUM_LANES = 31;
```

Comment:
Oh, but this is an app level constant though. I don't think it makes sense to put these in the canvas constants?

Reply:
I was confused about that myself. Maybe we can rename this to `appConstants.js` ? 
But if it's only a single constant in this file, we can add this to the top of `canvas/constant`.

Reply:
I think it's fine since they're 2 different domains. I am extremely opposed to putting this in canvas/constants.js as that defeats the purpose of splitting things into folders in the first place, and if we had to rename a file I'd rename the canvas one since these are global constants and are therefore more "authoritative". Either way I think this current set up is okay as the canvas constants file is located within the canvas folder and that folder can be viewed as a self-contained module of its own.

Reply:
Yeah, that's a valid point. It's good to go from my side!